stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@
 add_swift_target_library(swift_Concurrency ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_STDLIB
   Actor.swift
   PartialAsyncTask.swift
+  "${SWIFT_SOURCE_DIR}/stdlib/linker-support/magic-symbols-for-install-name.c"
```

Comment:
This isn't right -- `magic-symbols-for-install-name` only affects the install_name on operating systems that shipped without the Swift runtime, allowing the 5.0 compatibility libraries to continue to get picked up from the app bundle. This doesn't apply to libraries introduced since 5.0 shipped, like this one.

This is essentially a noop.

Reply:
Thanks for the explanation!

Addressed in 48a298fc676